nav2_planner/src/planner_server.cpp

```diff
@@ -241,11 +241,21 @@ PlannerServer::computePlan()
       return;
     }
 
+    // Copy needed as goal is const
```

Comment:
Remove comment, I think that's sufficiently clear. I think instead a comment above this block explaining its checking / changing the frames to `global_frame_` in case requests came in in another